engintron.sh

```diff
@@ -541,7 +541,8 @@ function chkserv_nginx_on {
         echo ""
         echo "=== Enable TailWatch chkservd driver for Nginx ==="
 
-        sed -i 's:service[httpd]=80,:service[httpd]=8080,:' /etc/chkserv.d/httpd
+        sed -i 's:service\[httpd\]=80,:service[httpd]=8080,:' /etc/chkserv.d/httpd
+        sed -i -e '$a\' /etc/chkserv.d/chkservd.conf
```

Comment:
What's this for?

Reply:
It adds a newline to the end of the file, if it doesn't exist, before appending "nginx:1". Before that, my chkservd.conf file was like this after engintron installation:
...
postgresql:1
rsyslogd:1
spamd:1
sshd:1nginx:1